pages/common/fakedata.md

```diff
@@ -0,0 +1,24 @@
+# fakedata
+
+> Generate fake data using a large variety of generators.
+> More information: <https://github.com/lucapette/fakedata>.
+
+- List all valid generators:
+
+`fakedata --generators`
+
+- Generate data using one or more generators:
+
+`fakedata {{generator1}} {{generator2}}`
+
+- Generate data with a specific output format:
+
+`fakedata --format {{csv|tab|sql}} {{generator}}`
+
+- Generate {{n}} data:
```

Comment:
n data sounds a bit weird. What is meant by one data?

Reply:
Well you know @marchersimon 'data' is plural for 'datum', hence 'n data' = 'n datums'.  I thought about this and decided to keep it as it's perfectly valid and integral English.

Reply:
Oh, my fault. I didn't know about `data` in that context. :laugh:

Reply:
What about?
```suggestion
- Generate data and limits rows up to {{n}} (default 10):
```
(I copied/pasted the usage 😇 / I'm not a native English speaker 😇)

Reply:
I think this should be with back tick instead of `{{`.

Reply:
>I think this should be with back tick instead of `{{`.

I thought about that, but I'm really not sure.  In many clients, wrapping it in `{{` and `}}` will highlight it, which we probably want.  @sbrl have you any opinions?

>What about? ...

@nicokosi How about this simpler version:
```suggestion
- Generate {{n}} rows of data (defaults to 10):
```

Reply:
Erm I don't think we've ever used the `{{token_syntax}}` in descriptions before. What about this?
```suggestion
- Generate a given number of fake data items (defaults to 10):
```

Reply:
Actually, we have in these pages:
- [`pages/linux/uvcdynctrl.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/linux/uvcdynctrl.md)
- [`pages/linux/cryptsetup.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/linux/cryptsetup.md)
- [`pages/linux/i3.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/linux/i3.md)
- [`pages/linux/trash.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/linux/trash.md)
- [`pages/common/nano.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/nano.md)
- [`pages/common/swig.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/swig.md)
- [`pages/common/bedtools.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/bedtools.md)
- [`pages/common/samtools.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/samtools.md)
- [`pages/common/dep.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/dep.md)
- [`pages/common/mosh.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/mosh.md)
- [`pages/common/ghdl.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/common/ghdl.md)
- [`pages/osx/whence.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages/osx/whence.md)
- [`pages.it/common/bedtools.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages.it/common/bedtools.md)
- [`pages.fr/common/ghdl.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages.fr/common/ghdl.md)
- [`pages.fr/common/nano.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages.fr/common/nano.md)
- [`pages.zh/linux/i3.md`](https://github.com/tldr-pages/tldr/blob/289787c7e8c1177742d23004198253154fe50c3c/pages.zh/linux/i3.md)

Reply:
> > I think this should be with back tick instead of `{{`.
> 
> I thought about that, but I'm really not sure. In many clients, wrapping it in `{{` and `}}` will highlight it, which we probably want. @sbrl have you any opinions?
> 
> > What about? ...
> 
> @nicokosi How about this simpler version:

Done in c687b5f9.